lib/Runtime/ByteCode/FuncInfo.cpp

```diff
@@ -12,25 +12,36 @@ FuncInfo::FuncInfo(
     Scope *bodyScope,
     ParseNode *pnode,
     Js::ParseableFunctionInfo* byteCodeFunction)
-    : alloc(alloc),
+    :
```

Comment:
super nitpick - the member initializer lists all use different code styles, maybe this PR is a good place to standardize at least the affected classes?

Reply:
Maybe, maybe not; it's a bit of a tangential change, and I'm not sure of the prevailing opinion on the issue (if there even is one).